app/views/staff/organisations/show.xml.haml

```diff
@@ -0,0 +1,7 @@
+!!! XML
+%iati-activities{"version" => "#{IATI_VERSION.gsub('_', '.')}",
+"generated-datetime" => "#{Time.now.strftime("%Y-%m-%dT%H:%M:%S")}"}
+  - @project_activities.each do |activity|
+    = render partial: "staff/shared/xml/activity", locals: { activity: activity, transactions: activity.transactions, budgets: activity.budgets }
+  - @third_party_project_activities.each do |activity|
```

Comment:
When this trello card was made I forgot that level D activities had been shipped. I think there might be a bit more logic involved here now based on the type of organisation. We should check this with Dan, I think it'll be manageable to deal with here 🤞 

I _think_ that:

- for UKSA only projects are published to IATI
- for Met Office and other NGO organisations _only_ third party projects are published to IATI
- there is no case when both levels should be published to IATI in the same file

I'll raise this with Dan on Slack and cc you in.

Reply:
https://dxw.slack.com/archives/CN0H2L066/p1589540222086000

Reply:
To close the loop on this for the record: Dan and Odeta confirmed this to be the shared understanding of how it should work. Resolving thread.